src/readme.graph.md

```diff
@@ -428,6 +428,13 @@ directive:
         let errorDetailsRegex = /(ErrorDetails\s*=\s*)(new.*ErrorDetails\(message\).*)/gmi
         $ = $.replace(errorDetailsRegex, '$1await this.GetErrorDetailsAsync((await response)?.Error, responseMessage)');
 
+        // Prevents null response objects to the output stream
+        let writeObjectRegex = /(WriteObject.*(await response).*;)/gm
+        var writeObjectRegexMatch = writeObjectRegex.exec($);
```

Comment:
The match should only happen for cases where response is a model type i.e., `global::System.Threading.Tasks.Task<Microsoft.Graph...PowerShell.Models.IMicrosoft...> response)`; otheriwise, the code won't compile when response is a non-nullable primitive type - https://microsoftgraph.visualstudio.com/Graph%20Developer%20Experiences/_build/results?buildId=122210&view=logs&j=534a3bb3-5058-5e77-0ce5-235e142aa1d6&t=34735c85-6252-5d84-51ff-ac0c0c134c95